tests/conftest.py

```
@@ -0,0 +1,51 @@
+    return repo_root / ".claude" / "skills"
+
+
+@pytest.fixture
+def hello_world_fixture():
+    """Get hello-world test fixture directory"""
+    return Path(__file__).parent / "fixtures" / "hello-world"
+
+
+def run_command(cmd, cwd=None, check=True):
+    """Helper to run shell commands"""
+    result = subprocess.run(
```

Ruff (0.14.10)

`subprocess` call with `shell=True` identified, security issue

(S602)